service/src/main/java/bio/terra/pipelines/app/configuration/external/IngressConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.pipelines.app.configuration.external;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "pipelines.ingress")
+public class IngressConfiguration {
+
+  /** Fully-qualified domain name. The base URL this instance can be accessed at. */
+  private String domainName;
```

Comment:
kinda funny this was already in our application.yml w/o us using it haha.

just to double check, does our helm-file overwrite this value when deploying to different environments?

Reply:
i think its actually more appropriate to have it called `tsps.ingress` so i would update the application.yml + the prefix here instead of the helm chart.

Also I dont think we're using the helm chart environment variable anywhere in our application.yml so we'll have to add that as well (no matter what we do with the naming)

Reply:
added the direct reference (though the spring magic did work without it)